spec/views/application/_flash_messages.html.erb_spec.rb

```diff
@@ -0,0 +1,25 @@
+require "rails_helper"
+
+RSpec.describe "rendering flash_messages" do
+  it "matches a notice message" do
+    flash[:notice] = "Testing flash notice"
+    render partial: "layouts/flash_messages"
+
+    expect(rendered).to match "Testing flash notice"
+    expect(rendered).to match "usa-alert--success"
+  end
+
+  it "matches an error message" do
+    flash[:error] = "Testing error notice"
+    render partial: "layouts/flash_messages"
+
+    expect(rendered).to match "Testing error notice"
+    expect(rendered).to match "usa-alert--error"
+  end
+
+  it "displays nothing if there are no messages" do
```

Comment:
[suggestion]: it'd be good to have a test for when there's both a flash error and notice, as there can be both at once!

Reply:
Good ideas!  I will do both